tests/test_preprocess_dsl.py

```diff
@@ -0,0 +1,533 @@
+from pathlib import Path
+
+import yaml
+from pytest_mock import MockerFixture
+
+import tested
+from tested.dsl.translate_parser import (
+    ExpressionString,
+    ReturnOracle,
+    _parse_yaml,
+    _validate_dsl,
+)
+from tested.nat_translation import (
+    convert_to_yaml,
+    create_enviroment,
+    parse_yaml,
+    run_translation,
+    to_yaml_object,
+    translate_yaml,
+    validate_pre_dsl,
+)
+
+test_unit_yaml_str = """
```

Comment:
These are quite long.
I would prefer separate files for any test yaml more then 10 lines.